README.md

```diff
@@ -46,7 +46,12 @@ Then run the tests and coverage report:
 If your build fails due to poor test coverage, *DO NOT* lower the test coverage threshold, instead inspect the generated report located here on your local repo: `/target/scala-2.12/scoverage-report/index.html`
 
 Then run the integration tests:
-NOTE: for integration tests to work make sure to execute command: `smserver` in a separate terminal window.
+
+NOTE: for integration tests to work, make sure to have all the external services required. 
+You can use the following commands to start/stop the required services.
+
+`sm2 --start DC_SECURE_MESSAGE_IT`
```

Comment:
Wes thinking more like:
`sm2 --start DC_SECURE_MESSAGE_IT`
`sbt it:test`
`sm2 --stop DC_SECURE_MESSAGE_IT`

Reply:
@ardwalker I think it is just to ensure all the services are running locally before running the it test. It is not the case that we have to run it all the time. We may already be having those services in local.

Reply:
Yes we only need to run it when we run the integration tests.